pkg/engine/template_generator.go

```diff
@@ -536,6 +536,14 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 			}
 			return false
 		},
+		"HasWindowsAgents": func() bool {
```

Comment:
there is already an existing function for this https://github.com/Azure/aks-engine/blob/master/pkg/api/types.go#L767

Reply:
and it doesn't look like you're using this anywhere right now, the e2e tests are using https://github.com/Azure/aks-engine/blob/master/test/e2e/engine/template.go#L198